test/inputs/goss.yaml

```diff
@@ -54,6 +54,27 @@ file:
     contents:
     - "#!/bin/bash"
 {{ end }}
+{{ if eq .Env.MACHINE_TYPE "machine" }}
+{{ if eq .Env.OS "almalinux" }}
+  "/etc/chrony.conf":
+    exists: true
+    contents:
+    - server custom.1.ntp.org prefer iburst
+{{ else }}
+  "/etc/systemd/timesyncd.conf":
+    exists: true
+    contents:
+    - NTP=custom.1.ntp.org
+  "/etc/systemd/resolved.conf.d/dns.conf":
+    exists: true
+    contents:
+    - DNS=1.1.1.1
+{{ end }}
+  "/etc/resolv.conf":
+    exists: true
+    contents:
+    - nameserver 1.1.1.1
```

Comment:
Why is this only for !machine ?

Reply:
I thought this way I can apply the test, to check if the file exist, for all machine types.